srml/authority-discovery/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "srml-authority-discovery"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "1.0.0", default-features = false, features = ["derive"] }
+sr-primitives = { path = "../../core/sr-primitives", default-features = false }
+primitives = { package = "substrate-primitives",  path = "../../core/primitives", default-features = false }
+app-crypto = { package = "substrate-application-crypto", path = "../../core/application-crypto", default-features = false }
+rstd = { package = "sr-std", path = "../../core/sr-std", default-features = false }
+serde = { version = "1.0", optional = true }
+session = { package = "srml-session", path = "../session", default-features = false }
+im-online = { package = "srml-im-online", path = "../im-online", default-features = false }
+srml-support = { path = "../support", default-features = false }
+sr-io = { package = "sr-io", path = "../../core/sr-io", default-features = false }
+system = { package = "srml-system", path = "../system", default-features = false }
+
+[features]
+default = ["std", "session/historical"]
```

Comment:
Don't you want to have the historical feature in wasm as well?

Reply:
Actually this was a left over from some testing. None of `authority-discovery` depends on `session/historical` thus I removed the reference.